exchanges/kucoin/kucoin_test.go

```diff
@@ -1999,6 +2001,20 @@ func verifySubs(tb testing.TB, subs subscription.List, a asset.Item, prefix stri
 	}
 }
 
+var subPairs currency.Pairs
+
+func init() {
+	for _, pp := range [][]string{
+		{"BTC", "USDT", "-"}, {"ETH", "BTC", "-"}, {"ETH", "USDT", "-"}, {"LTC", "USDT", "-"}, // Spot
+		{"ETH", "BTC", "-"}, {"LTC", "USDT", "-"}, {"SOL", "USDC", "-"}, {"TRX", "BTC", "-"}, // Margin
+		{"ETH", "USDCM", ""}, {"SOL", "USDTM", ""}, {"XBT", "USDCM", ""}, // Futures
+	} {
+		pair := currency.NewPairWithDelimiter(pp[0], pp[1], pp[2])
+		pair.Delimiter = pp[2]
+		subPairs = append(subPairs, pair)
```

Comment:
```suggestion
		subPairs = append(subPairs, currency.NewPairWithDelimiter(pp[0], pp[1], pp[2]))
```

Reply:
Yeah. That's some first class code rot there.

Fixed 64ba0b3